Sources/CXExtensions/Await.swift

```diff
@@ -0,0 +1,138 @@
+import CXShim
+import Foundation
+import CoreFoundation
+
+extension Publisher {
+    
+    public func sync() -> Subscribers.Await<Output, Failure> {
```

Comment:
Naming suggestion?

candidate:
- synchronized
- wait
- blocking

Reply:
Sorry for the late reply, well, `wait` is clearer maybe?

```swift
// NSCondition
condition.wait()

// Operation
op.wait()

// DispatchGroup
g.wait()
```

Reply:
This operation does not actually "wait" anything until you iterate it.
```swift
let sub = pub.wait() // non blocking
sub.next() // block here
```

Reply:
I take `Blocking` from [RXBlocking](https://github.com/ReactiveX/RxSwift/tree/master/RxBlocking)

Reply:
make sense